loggable.go

```diff
@@ -42,6 +42,8 @@ type ChangeLog struct {
 	ObjectType string      `gorm:"index"`
 	RawObject  string      `sql:"type:JSON"`
 	RawMeta    string      `sql:"type:JSON"`
+	RawDiff    string      `sql:"type:JSONB"`
```

Comment:
It's better to use ANSI JSON, because jsonb it's only postgresql feature.

Reply:
Sure. Just missed that during the merge. :)